src/packages/excalidraw/CHANGELOG.md

```diff
@@ -23,6 +23,12 @@ Please add the latest change on the top under the correct section.
 
   This also removes the GitHub icon, keeping it local to the https://excalidraw.com app.
 
+### Fixes
+
+- The encryption shield icon is now removed from excalidraw package as it was specific to excalidraw app and rendered via [`renderFooter`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#renderFooter) prop. In case you were hiding this icon earlier, you need not do that anymore [#3577](https://github.com/excalidraw/excalidraw/pull/3577).
```

Comment:
Was it rendered via `renderFooter`? I think it wasn't, which was the point of this PR?

Reply:
Yes, I will better update it to `and now rendered via `renderFooter` prop to avoid the confusion

Reply:
Actually it was already updated, you can check https://github.com/excalidraw/excalidraw/pull/3577/commits/b6e7a9f6dc85f647eaa6180b656275369db990ae

Reply:
oh, I guess GitHub showed me an old commit, sorry!

Reply:
Merge whenever you're ready!

Reply:
Actually, haven't reviewed readme yet. Gimme a min!

Reply:
Done!

Reply:
Thank you 👍🏻